master/internal/db/postgres_helpers_intg_test.go

```diff
@@ -154,3 +154,20 @@ func requireMockMetrics(
 	require.NoError(t, err)
 	return &m
 }
+
+func requireMockCheckpoint(
+	t *testing.T, db *PgDB, tr *model.Trial, latestBatch int,
+) *trialv1.CheckpointMetadata {
+	ckpt := trialv1.CheckpointMetadata{
+		TrialId:           int32(tr.ID),
+		Uuid:              uuid.NewString(),
+		Resources:         map[string]int64{"ok": 1.0},
+		Framework:         "some framework",
+		Format:            "some format",
+		DeterminedVersion: "1.0.0",
+		LatestBatch:       int32(latestBatch),
+	}
+	err := db.AddCheckpointMetadata(context.TODO(), &ckpt)
```

Comment:
I don't think this is the signature of this method, anymore. if you look at `postgres_model_intg_test.go`, it does
```
			ckpt := &model.CheckpointV2{
				UUID:         uuid.New(),
				TaskID:       tr.TaskID,
				AllocationID: a.AllocationID,
				ReportTime:   time.Now().UTC(),
				State:        model.CompletedState,
				Resources: map[string]int64{
					"ok": 1.0,
				},
				Metadata: map[string]interface{}{
					"framework":          "some framework",
					"format":             "some format",
					"determined_version": "1.0.0",
					"steps_completed":    stepsCompleted,
				},
			}
			err = db.AddCheckpointMetadata(context.TODO(), ckpt)
```

Reply:
could define this in `postgres_test_utils` and use it there and where ^ that snippet is